src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    protected Object clone() throws CloneNotSupportedException {
+        return super.clone();
+    }
```

Comment:
Same, do we need this method?

Reply:
It was auto generated.
It isn't necessarily method.
I deleted it.